src/pages/admin-ui-sdk/publish.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Prepare your app for production
+description: Learn how to prepare your app for non-local testing and for production.
+keywords:
+  - App Builder
+  - Extensibility
+---
+
+# Prepare your app for production
+
+After you have tested your app locally, you are ready to publish your app to your Production environment. When you have accomplished this, your app is available in the Adobe App Registry. Adobe Commerce will then be able fetch the information it needs to augment the Admin.
+
+During the testing phase, an administrator of your enterprise organization performs the approvals. At this point, your app is not public. When you are ready to make the app available publicly, submit the app to the Adobe Review team instead of your admin.
+
+[Publishing Your First App Builder Application](https://developer.adobe.com/app-builder/docs/getting_started/publish_app/) in the _App Builder Getting Started_ guide describes how to get your app ready for production. Deploy your app to your Production workspace, then use the following steps:
+
+1. Submit for publishing approval on your Adobe Developer Console.
+
+1. Your app must be approved by an enterprise organization administrator. Your administrator uses the My Exchange panel in the Developer Console to perform this task. Upon approval, the app is published and available on Experience Cloud. Adobe Commerce can now find the app in the App Registry.
+
+1. Perform all testing. If any changes need to be made to your app, ask the administrator to reject the app within the My Exchange utility. Update your code and resubmit your app for approval. Repeat these steps until you have completed all testing.
+
+1. When you have completed all testing and are ready to publicly publish the app, submit your app to [Commerce Marketplace](https://developer.adobe.com/commerce/marketplace/guides/sellers/apps/).
```

Comment:
Let's remove this line - the reference to a separate and unrelated submission process for Commerce Marketplace is confusing